Task 5.1/Task5.1/SparseMatrix.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+
+namespace Task5._1
+{
```

Comment:
Not effective, this is not a sparse matrix. You allocate memory for long[rows, columns], to store only 2 non-zero values, for example. Please think to store only non-zero values inside, and provide them by indexer (idea is similar to Diagonal matrix)

Reply:
Fixed